CHANGES_NEXT_RELEASE

```diff
@@ -3,3 +3,4 @@
 - Add: Forwarded updates work for registrations using idPattern == '.*', but only for simple cases (#3458)
```

Comment:
The functionality should be described in the documentation. However, let's defer that to a new PR, which I will do. For this one, please just include a section in https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/user/ngsiv2_implementation_notes.md just after "Oneshot subscriptions" with a mark like "TBD: to be completed by Fermin".

Reply:
I think this comment is yet pending...

Reply:
fixed in 27eb837